src/screens/NewAnalytics/PaymentAnalytics/PaymentsDistribution/PaymentsDistributionTypes.res

```diff
@@ -0,0 +1,8 @@
+type queryData = PaymentsSuccessRate | Connector
+
+type categories = [#connector]
```

Comment:
use can use `dimention` type from `NewAnalyticsTypes` both are the same

Reply:
okay